Download.Mvc/AppSettings.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Box.V2;
+using Box.V2.Auth;
+using Box.V2.Config;
+using Box.V2.Models;
```

Comment:
@laholstege doesn't hurt anything but you only need the first of these using statements, you aren't using anything else in here